frontend/packages/kubevirt-plugin/src/components/vm-disks/_disk-summary.scss

```diff
@@ -0,0 +1,10 @@
+.kubevirt-disk-summary-disk {
+  margin-bottom: 10px;
+
+  .kubevirt-disk-summary-disk-title {
+    font-weight: 600;
```

Comment:
Isn't there a constant for that? To stay consistent ... 
Cc @matthewcarleton

Reply:
@glekner has this been removed? I don't see the file anymore 🤔

Reply:
@matthewcarleton filename changed to `disk-summary.scss`.
This file resembles the details view of the cds, not the modal